examples/deferred/main.rs

```diff
@@ -84,7 +86,7 @@ struct CubeVertex {
     pos: [i8; 3],
 }
 
-#[shader_param]
+#[shader_param(R)]
```

Comment:
Right, I just wasn't able to come up with anything better than that... The problem comes from the fact that this structure has to implement `ShaderParam` for every device. We need to somehow make it device-independent.